pxtrunner/renderer.ts

```diff
@@ -1261,9 +1261,29 @@ function renderGhost(options: ClientRenderOptions) {
     c.remove();
 }
 
+function renderTemplate(options: ClientRenderOptions) {
+    let c = $('code.lang-template');
+    if (options.snippetReplaceParent)
```

Comment:
Thanks for the advice! When I did this, it also made me realize that it would look cleaner if I moved the `render` function that both the scoped renders use out to have less repeated code.